src/main/scala/io/iohk/ethereum/blockchain/sync/FastSync.scala

```diff
@@ -168,10 +173,16 @@ class FastSync(
     }
 
     def waitingForTargetBlockUpdate(processState: FinalBlockProcessingResult): Receive = handleCommonMessages orElse {
-      case FastSyncTargetBlockSelector.Result(targetBlockHeader) =>
-        log.info(s"new target block with number ${targetBlockHeader.number} received")
+      case FastSyncTargetBlockSelector.Result(targetBlockHeader, mostUpToDateBlockHeader) =>
+        log.info(s"new target block with number ${targetBlockHeader.number} received, new most up to date block number is ${mostUpToDateBlockHeader.number}")
```

Comment:
Here it should not have a big impact but in general it's advised to log in this manner:
```scala
log.info("new target block with number {} received, new most up to date block number is {}", targetBlockHeader.number, mostUpToDateBlockHeader.number)
```
In this way string template is being filled only if given message is going to be appended to log. Here's short article about that: https://manuel.bernhardt.io/2016/08/31/akka-anti-patterns-logging-the-wrong-way/

Reply:
I know that :) But this log has an `info` level. It will be evaluated always. I could change that for consistency.